Multiple Changes: Remove eps_modified variable and add EPS_MODIFIED HondaFlag, change format of Clarity torqueBP values from hexadecimal to decimal, update Honda Clarity torque controller params to auto-tuned values instead of a copy of Civic values

roenthomas opened the pull request:
Evaluated values are unchanged.

"Kiril — 9/16/2021 2:27 PM
It would be so much easier if people just used integers for both instead of hex"

Clarity is the only car in the Honda interface.py that uses heaxdecimal modded torqueBP values; stock torqueBP values are in decimal and all other cars' stock and modded torqueBP values are also in decimal.

This PR brings Clarity in line with all the other Hondas.

roenthomas committed:
Change format of Clarity torqueBP values from hexadecimal to decimal

Evaluated values are unchanged.

"Kiril — 9/16/2021 2:27 PM
It would be so much easier if people just used integers for both instead of hex"

FrogAi commented:
Thank you for your PR! If you're not already in the FrogPilot Discord, [feel free to join](https://discord.FrogPilot.download) and let me know you've opened a PR!

roenthomas committed:
Merge branch 'FrogAi:MAKE-PRS-HERE' into MAKE-PRS-HERE

roenthomas committed:
Merge branch 'FrogAi:MAKE-PRS-HERE' into MAKE-PRS-HERE

roenthomas committed:
Merge branch 'FrogAi:MAKE-PRS-HERE' into MAKE-PRS-HERE

roenthomas committed:
Merge branch 'FrogAi:MAKE-PRS-HERE' into MAKE-PRS-HERE

roenthomas committed:
Updated Honda Values with new flags to support eps_modified and remove eps_modified variable in favor of HondaFlags-based approach

Integrated twilsonco commit https://github.com/twilsonco/sunnypilot/commit/224f07501beb10f7748b6b23ff7670f54a456b93

Can access the value of ret.flags outside of interface.py

updated selfdrive/car/honda/values.py and interface.py

Co-Authored-By: Tim Wilson <7284371+twilsonco@users.noreply.github.com>

roenthomas committed:
Update params.toml with auto-tuned values for friction and lat accel factor for stock Clarity

Existing values for HONDA_CLARITY are a copy of HONDA_CIVIC, which is incorrect.

roenthomas committed:
Update ki in interfaces.py to match twilsonco recommended value

As per @twilsonco
"If there's no oscillations, then it's not too high.

For the longest time, ki was kept very low due to oscillations. When the torque controller was introduced, for ki to be 10% of kp was a rather high value."

Changes in this commit already present here:
https://github.com/FrogAi/FrogPilot/blob/0c0c46af/selfdrive/car/interfaces.py

Co-Authored-By: Tim Wilson <7284371+twilsonco@users.noreply.github.com>

roenthomas committed:
Revert "Update ki in interfaces.py to match twilsonco recommended value"

This reverts commit fbd54465cd0f8e39862b420cb9288bdbfcb9c000.

roenthomas committed:
Merge branch 'FrogAi:MAKE-PRS-HERE' into MAKE-PRS-HERE
